rosidl_typesupport_c/rosidl_typesupport_c-extras.cmake.in

```diff
@@ -10,6 +10,8 @@ ament_register_extension(
   "rosidl_typesupport_c"
   "rosidl_typesupport_c_generate_interfaces.cmake")
 
+set(rosidl_typesupport_c_SUPPORT_POCO @Poco_FOUND@)
```

Comment:
I'd recommend `_SUPPORTS_POCO` or `_HAS_POCO` or `_SUPPORTS_MULTIPLE_RMW`. `_SUPPORT_POCO` reads like the answer to a question:

- Does `rosidl_typesupport_c` support Poco? Yes **`rosidl_typesupport_c` support poco**.

And that doesn't sound right. I suppose it makes sense if you style it as an order: "`rosidl_typesupport_c`, support Poco." But I think this is an answer to a question not an order or directive.

(same for cpp version)